--- cmake/base.cmake.orig	2019-06-23 15:06:08.415173847 +0200
+++ cmake/base.cmake	2019-06-23 15:08:18.866267616 +0200
@@ -40,13 +40,13 @@
 
 # add install prefix to pkgconfig search path if needed
 string (REGEX REPLACE "([\\+\\(\\)\\^\\\$\\.\\-\\*\\?\\|])" "\\\\\\1" PKGCONFIG_REGEX ${CMAKE_INSTALL_PREFIX})
-set (PKGCONFIG_REGEX ".*${PKGCONFIG_REGEX}/lib/pkgconfig:${PKGCONFIG_REGEX}/share/pkgconfig.*")
+set (PKGCONFIG_REGEX ".*${PKGCONFIG_REGEX}/lib${LIB_SUFFIX}/pkgconfig:${PKGCONFIG_REGEX}/share/pkgconfig.*")
 
 if (NOT "$ENV{PKG_CONFIG_PATH}" MATCHES "${PKGCONFIG_REGEX}")
     if ("" STREQUAL "$ENV{PKG_CONFIG_PATH}")
-	set (ENV{PKG_CONFIG_PATH} "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
+	set (ENV{PKG_CONFIG_PATH} "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
     else ()
 	set (ENV{PKG_CONFIG_PATH}
-	    "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig:$ENV{PKG_CONFIG_PATH}")
+	    "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig:$ENV{PKG_CONFIG_PATH}")
     endif ()
 endif ()
