--- utsushi-0.65.0/sane/Makefile.am.orig	2022-07-28 09:19:26.418665383 +0200
+++ utsushi-0.65.0/sane/Makefile.am	2022-07-28 09:42:12.682519616 +0200
@@ -216,28 +216,12 @@
 	  echo " skipping component activation"; \
 	  exit 1; \
 	fi
-	if test -d "$(SANE_SYSCONFDIR)/dll.d"; then \
-	  dirname="$(DESTDIR)$(SANE_SYSCONFDIR)/dll.d"; \
-	  file=$$dirname/$(BACKEND_NAME); \
-	  test -d $$dirname || $(MKDIR_P) $$dirname; \
-	  echo "#  $(BACKEND_NAME) -- dll.conf snippet" > $$file; \
-	  echo "" >> $$file; \
-	  echo "$(BACKEND_NAME)" >> $$file; \
-	else \
-	  dirname="$(DESTDIR)$(SANE_SYSCONFDIR)"; \
-	  file=$$dirname/dll.conf; \
-	  if test -f $$file; then \
-	    if test -n "`$(GREP) '^[ \t]*#[ \t#]*$(BACKEND_NAME)' $$file`"; \
-	    then \
-	      sed -i 's,^[ \t]*#[ \t#]*\($(BACKEND_NAME)\),\1,' $$file; \
-	    elif test -z "`$(GREP) $(BACKEND_NAME) $$file`"; then\
-	      echo $(BACKEND_NAME) >> $$file; \
-	    fi; \
-	  else \
-	    echo $(ECHO_N) "WARNING: $$file not found,"; \
-	    echo " cannot activate SANE component"; \
-	  fi; \
-	fi
+	dirname="$(DESTDIR)$(SANE_SYSCONFDIR)/dll.d"; \
+	file=$$dirname/$(BACKEND_NAME).conf; \
+	test -d $$dirname || $(MKDIR_P) $$dirname; \
+	echo "#  $(BACKEND_NAME) -- dll.conf snippet" > $$file; \
+	echo "" >> $$file; \
+	echo "$(BACKEND_NAME)" >> $$file;
 
 uninstall-hook-data:
 	@$(NORMAL_UNINSTALL)
