--- a/security/Kconfig	2021-12-05 18:20:55.655677710 -0500
+++ b/security/Kconfig	2021-12-05 18:23:42.404251618 -0500
@@ -167,6 +167,7 @@ config HARDENED_USERCOPY_PAGESPAN
 	bool "Refuse to copy allocations that span multiple pages"
 	depends on HARDENED_USERCOPY
 	depends on BROKEN
+	depends on !GENTOO_KERNEL_SELF_PROTECTION
 	help
 	  When a multi-page allocation is done without __GFP_COMP,
 	  hardened usercopy will reject attempts to copy it. There are,
