diff --git a/security/Kconfig b/security/Kconfig
index 7561f6f99..01f0bf73f 100644
--- a/security/Kconfig
+++ b/security/Kconfig
@@ -166,5 +166,6 @@ config HARDENED_USERCOPY
 config HARDENED_USERCOPY_FALLBACK
 	bool "Allow usercopy whitelist violations to fallback to object size"
 	depends on HARDENED_USERCOPY
+	depends on !GENTOO_KERNEL_SELF_PROTECTION
 	help
 	  This is a temporary option that allows missing usercopy whitelists
