--- src/lib/Sympa/DatabaseDriver/SQLite.pm.old	2016-01-06 11:18:57.000000000 +0100
+++ src/lib/Sympa/DatabaseDriver/SQLite.pm	2016-12-04 17:03:15.872560407 +0100
@@ -455,7 +455,7 @@
         $param->{'index'}, $param->{'table'});
 
     my $sth;
-    unless ($sth = $self->do_query(q{DROP INDEX "%s"}, $param->{'index'})) {
+    unless ($sth = $self->do_query(q{DROP INDEX IF EXISTS "%s"}, $param->{'index'})) {
         $log->syslog('err',
             'Could not drop index %s from table %s in database %s',
             $param->{'index'}, $param->{'table'}, $self->{'db_name'});
@@ -806,7 +806,7 @@
     } elsif ($r) {
         return $r;
     } else {
-        unless ($self->do_query(q{DROP TABLE "%s"}, $table)) {
+        unless ($self->do_query(q{DROP TABLE IF EXISTS "%s"}, $table)) {
             $log->syslog('err', 'Could not drop table "%s"', $table);
             return undef;
         }
