--- src/sympa_wizard.pl.orig	2007-03-04 11:41:19.000000000 +0100
+++ src/sympa_wizard.pl	2007-09-16 19:24:00.377085123 +0200
@@ -92,13 +92,13 @@
                'advice' =>''},
 
 	      {'name' => 'arc_path',
-	       'default' => '--DIR--/arc',
+	       'default' => '--SPOOLDIR--/arc',
 	       'query' => 'Where to store HTML archives',
 	       'file' => 'wwsympa.conf','edit' => '1',
                'advice' =>'Better if not in a critical partition'},
 	      
 	      {'name' => 'bounce_path',
-	       'default' => '--DIR--/bounce',
+	       'default' => '--SPOOLDIR--/bounce',
 	       'query' => 'Where to store bounces',
 	       'file' => 'wwsympa.conf',
                'advice' =>'Better if not in a critical partition'},
@@ -122,13 +122,13 @@
 	       'advice' =>''},
 	      
 	      {'name' => 'queuebounce',
-	       'default' => '--SPOOLDIR--/bounce',
+	       'default' => '--SPOOLDIR--/qbounce',
 	       'query' => 'Bounce incoming spool',
 	       'file' => 'sympa.conf',
 	       'advice' =>''},
 
 	      {'name' => 'static_content_path',
-	       'default' => '--DIR--/static_content',
+	       'default' => '--SPOOLDIR--/static_content',
 	       'query' => 'The directory where Sympa stores static contents (CSS, members pictures, documentation) directly delivered by Apache',
 	       'file' => 'sympa.conf',
 	       'advice' =>''},	      
--- wwsympa/Makefile.orig	2008-01-30 18:10:56.000000000 +0100
+++ wwsympa/Makefile	2008-04-02 09:26:53.222445659 +0200
@@ -100,8 +100,8 @@
 
 installicons:
 	@echo "Installing icons..."
-	@echo $(DESTDIR)$(DIR)/static_content/icons
-	cp icons/*.png $(DESTDIR)$(DIR)/static_content/icons
+	@echo $(DESTDIR)$(SPOOLDIR)/static_content/icons
+	cp icons/*.png $(DESTDIR)$(SPOOLDIR)/static_content/icons
 
 installwrapper:
 	@echo "Installing wwsympa-wrapper binary..."
