--- src/sympa.pl.orig	2008-04-25 11:14:22.778440706 +0200
+++ src/sympa.pl	2008-04-25 11:15:04.042853056 +0200
@@ -203,6 +203,7 @@
 ## Set locale configuration
 $main::options{'lang'} =~ s/\.cat$//; ## Compatibility with version < 2.3.3
 $Language::default_lang = $main::options{'lang'} || $Conf{'lang'};
+&Language::SetLang($Language::default_lang);
 
 ## Main program
 if (!chdir($Conf{'home'})) {
@@ -827,8 +828,6 @@
 ## them, sleeps a while and continues the good job.
 while (!$signal) {
 
-    &Language::SetLang($Language::default_lang);
-
     &List::init_list_cache();
 
     if (!opendir(DIR, $spool)) {
