--- src/dbiquote.h.orig	2007-04-25 14:08:10.000000000 +0200
+++ src/dbiquote.h	2012-01-06 10:11:22.358697110 +0100
@@ -25,6 +25,7 @@
 #endif
 
 #include <string>
+#include <cstring>
 #include <stdexcept>
 
 #define DBIQUOTEDBG false
--- src/main.cpp.orig	2007-04-25 14:08:10.000000000 +0200
+++ src/main.cpp	2012-01-06 10:12:15.815189588 +0100
@@ -28,6 +28,7 @@
 #include <fstream>
 #include <sstream>
 #include <syslog.h>
+#include <unistd.h>
 #include "defs.h"
 #include "triplet.h"
 #include "read.h"
@@ -110,7 +110,7 @@
 	}
 }
 /// main
-int main(char argc,char* argv[])
+int main(int argc,char* argv[])
 {
 //	Watchdog harakiri;
 	openlog(::s_gpsname, LOG_PID, GPS_LOG_FACILITY);
--- src/db.cpp.orig	2012-01-06 10:10:17.266090487 +0100
+++ src/db.cpp	2012-01-06 10:13:08.947679800 +0100
@@ -31,6 +31,7 @@
 #include "triplet.h"
 #include <sstream>
 #include <stdexcept>
+#include <algorithm>
 #include "cfg.h"
 #include "tmplwlmod.h"
 #include "wldb.h"
--- src/wldb.h.orig	2007-04-25 14:08:10.000000000 +0200
+++ src/wldb.h	2012-01-06 10:13:37.663941692 +0100
@@ -25,6 +25,7 @@
 */
 #include <exception>
 #include <string>
+#include <cstring>
 #include <syslog.h>
 #include "triplet.h"
 #include "db.h"
--- src/triplet.h.orig	2007-04-25 14:08:10.000000000 +0200
+++ src/triplet.h	2012-01-06 10:17:47.194249074 +0100
@@ -25,6 +25,7 @@
 #include <config.h>
 #endif
 #include <iostream>
+#include <stdlib.h>
 #include <string>
 #include <sstream>
 #include <stdexcept>
